stock_available/README.rst

```diff
@@ -0,0 +1,77 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Could you please follow migration guide about readme fragments : https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0 ?